pkgs/development/python-modules/cairosvg/default.nix

```diff
@@ -59,7 +57,7 @@ buildPythonPackage rec {
 
   meta = {
     homepage = "https://cairosvg.org";
-    changelog = "https://github.com/Kozea/CairoSVG/releases/tag/${version}";
+    changelog = "https://github.com/Kozea/CairoSVG/releases/tag/${src.tag}";
```

Comment:
IMO it's better to have a little repetition (e.g: write `v${version}` in `src` and `meta`), rather than use `src.tag` and run into #367739.

`buildPython{Package,Application}` don't yet support `finalAttrs`, but I'd still err on the side of caution so that there is not risk of running into this issue in the future when it does.